### ISSUE TITLE
ImportError: No module named 'inplace_abn'

### ISSUE THREAD

**Issue by shanjiuvspikaqiu** [juntang-zhuang/ShelfNet#22]
Thank you for the great work! I met a problem with  **No module named 'inplace_abn'** when I run _python3 evaluate.py_. After installing inplace_abn from official repo, it works still not. 

Do you have any advices? THX and have a nice day!

**Comment by juntang-zhuang** [comment]
Hi, I'm not sure, could it be a problem related to the path in your python environment? The source of inplace_abn is in https://github.com/juntang-zhuang/ShelfNet/tree/941c71cc85a3d6227c03ed435b1ad80c6c39573f/ShelfNet18_realtime/modules/src  Could you try setting up from here?

**Comment by shanjiuvspikaqiu** [comment]
@juntang-zhuang Thank you for the reply. I will be appreciate, if you could tell me how to set up from /moudules/src.

**Comment by juntang-zhuang** [comment]
I can directly run it. I guess it's a problem with your path environment. My code is modified from https://github.com/CoinCheung/BiSeNet/tree/master/old , seems no need to setup

**Comment by shanjiuvspikaqiu** [comment]
@juntang-zhuang You are right, thank you! I changed the environment into **Python 3.6 and PyTorch 1.1** , it works! 
(tiny suggestion: maybe you can change the Requirements more specific in README.md)

I tried python 3.7 & PyTorch 1.2, 1.6, 1.7, found you mentioned here https://github.com/juntang-zhuang/ShelfNet/issues/18#issuecomment-689057199
Are there any codes to inference the images? I'd like to see how good the result with rgb labels is. Have a nice day!

**Comment by juntang-zhuang** [comment]
I think you can change the dataloader to inference the images. Currently I don't have any code to directly inference on given images. 
Just changed readme with PyTorch version, thanks for suggestion.
